GamebookEngine/Views/Editing/Page List/PagesTableViewController.swift

```diff
@@ -38,6 +41,9 @@ class PagesTableViewController: UITableViewController {
 
         tableView.register(UINib(nibName: "PageTableViewCell", bundle: nil), forCellReuseIdentifier: "pageCell")
 
+        pagesSearchBar.delegate = self
+        pagesSearchBar.returnKeyType = .done
```

Comment:
It behaves similarly to the default behavior. It just changes the text on the blue button for the iOS keyboard. Honestly, line 45 can be removed.

Reply:
Changing this to `.search` could be good, even if it doesn't do anything, at least it'd look nice? `.done` should dismiss the keyboard and erase the search results, I'd think.

Reply:
I believe ```.search``` is done by default. Thus, ```pagesSearchBar.returnKeyType``` can be omitted.

Reply:
Hmm, so once I got it up on my device I guess `.search` is weird, 'cause it just closes the keyboard. So maybe your original `.done` was better. There's definitely a lot of weirdness around what to do with the keyboard and search results when `searchBarSearchButtonClicked` is triggered. Since searching is happening automatically, I'd vote for `.done` and to have `searchBarSearchButtonClicked` clear out the search.

Reply:
This is more of a design question, but do you think we should dismiss the search keyboard when the user starts scrolling through search results? On small displays, or on an iPad Air in landscape, it's a little crammed to scroll through the search results with the keyboard up. Would be nice to hide it I think?

Reply:
Hiding it would be good when scrolling, especially for smaller devices.

Reply:
I overriden the scrollViewWillBeginDragging method so that the keyboard is dismissed when the user scrolls through the table view.

Reply:
Alternately you could do this, since we're using XIBs in this project:

https://stackoverflow.com/a/49885521/2117288

It doesn't really change the feel/functionality of this, this is just an FYI.

Reply:
The latest commit should resolve this issue.